.github/workflows/end2end.yaml

```diff
@@ -159,6 +159,30 @@ jobs:
           password: ${{ secrets.ARTIFACTS_PASSWORD }}
           source: /tmp/artifacts
         if: always()
+    
+  check-mongo-patches:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: remove existing tar file
```

Comment:
no point, they are not commited
(and consequently, not sure the makefile rule is really needed...)